cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -82,16 +82,25 @@ export function computeZRange(states: any[]): number[] {
     return [minZ, maxZ];
 }
 
+interface ILabelType extends Object {
+    total: number
+}
+
+function getMetricTags(labels: Record<string, ILabelType>): Record<string, number> {
```

Comment:
If `labels` is unitialized it will be `undefined` and `Object.entries(labels)` is going to throw an Exception. We can guard ourselves by assigning a default value on args:
```suggestion
function getMetricTags(labels: Record<string, ILabelType> = {}): Record<string, number> {
```